src/electron/routing_service.ts

```diff
@@ -323,7 +323,7 @@ async function installLinuxRoutingServices(): Promise<void> {
   command +=
     ' && ' +
     installationFileDescriptors
-      .map(({filename, sha256}) => `/usr/bin/echo "${sha256}  ${path.join(tmp, filename)}" | /usr/bin/shasum -a 256 -c`)
+      .map(({filename, sha256}) => `/usr/bin/echo "${sha256}  ${path.join(tmp, filename)}" | $(which shasum) -a 256 -c`)
```

Comment:
We can't use the path because it opens up  a security vulnerability, that's why we use absolute paths to read-only files.
One alternative would be to check a few absolute paths instead.

Reply:
/cc @jyyi1